gcovr/configuration.py

```diff
@@ -807,14 +807,14 @@ def merge_options_and_set_defaults(
         "show_decision",
         ["--decisions"],
         group="output_options",
-        help="Report the decision coverage. For HTML and JSON report.",
+        help="Report the decision coverage. For HTML, JSON  and the summary report.",
```

Comment:
```suggestion
        help="Report the decision coverage. For HTML, JSON, and the summary report.",
```

Reply:
This additional coma is wrong.

Reply:
It's and oxford comma, it's a stylistic choice so entirely up to you. I suggested adding it because I saw it used elsewhere in gcovr documentation, e.g. https://github.com/gcovr/gcovr/blob/2591d80014a2f7bb681863d1adf978c12d8560b0/gcovr/configuration.py#L1209

Reply:
Didn't know the "Oxford comma", learned something new.